ui-participant/src/landing/registration/ResetPasswordPage.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react'
+import { useSearchParams } from 'react-router-dom'
+import { useAuth } from 'react-oidc-context'
+import { PageLoadingIndicator } from 'util/LoadingSpinner'
+import { getEnvSpec } from 'api/api'
+import { getB2CLocale } from '@juniper/ui-core'
+
+/** Page for participants who already have enrollee data in Juniper (from a migration or admin action), and need
+ * to join to link their account */
+export default function ResetPasswordPage() {
+  const auth = useAuth()
+  const [searchParams] = useSearchParams()
+  const accountName = searchParams.get('accountName') || ''
+  const preferredLanguage = searchParams.get('preferredLanguage') || ''
+  const envSpec = getEnvSpec()
+
+  const createAccount = async () => {
+    auth.signinRedirect({
+      redirectMethod: 'replace',
+
+      extraQueryParams: {
+        option: 'resetpassword',
```

Comment:
this `option` only works because of the corresponding terraform-ap-deployments PR; we already hade `option: signup`, so it's set up similarly.